README.md

```diff
@@ -1,3 +1,85 @@
-# Futbol
+# Futbol :soccer:
 
-Starter repository for the [Turing School](https://turing.io/) Futbol project..
+## Table of Contents
```

Comment:
This looks great!

Reply:
Thanks! The table of contents might be a bit much for our project but I threw it in anyway.

Reply:
Wow this is a phenomenal start!

Reply:
Thanks!
I googled great personal project readme template and a [Turing template](https://gist.github.com/martensonbj/6bf2ec2ed55f5be723415ea73c4557c4) came up first lol.